packages/office-ui-fabric-react/src/components/Dialog/Dialog.base.tsx

```diff
@@ -51,7 +52,7 @@ export class DialogBase extends BaseComponent<IDialogProps, {}> {
       isBlocking: 'modalProps.isBlocking',
       containerClassName: 'modalProps.containerClassName',
       onDismissed: 'modalProps.onDismissed',
-      onLayerDidMount: 'modalProps.onLayerDidMount',
+      onLayerDidMount: 'modalProps.layerProps.onLayerDidMount',
```

Comment:
nit: Update the deprecated message for the JSDocs in [here](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/Dialog/Dialog.types.ts#L95) to `modalProps.layerProps.onLayerDidMount` to match this warning.